homeassistant/components/toon/__init__.py

```diff
@@ -80,7 +80,7 @@ async def async_setup_entry(hass: HomeAssistantType, entry: ConfigType) -> bool:
     )
     hass.data.setdefault(DATA_TOON_CLIENT, {})[entry.entry_id] = toon
 
-    toon_data = ToonData(hass, entry, toon)
+    toon_data = await hass.async_add_executor_job(ToonData, hass, entry, toon)
```

Comment:
Side effects in init method. :see_no_evil:

Reply:
This thing is going to be replaced in total for 0.110 or 0.111...